conversion/conversion.go

```diff
@@ -74,19 +74,22 @@ var (
 )
 
 // SchemaConv performs the schema conversion
+// TODO: Pass around cmd.SourceProfile instead of sqlConnectionStr and schemaSampleSize.
+// Doing that requires refactoring since that would introduce a circular dependency between
+// conversion.go and cmd/source_profile.go.
 // The sqlConnectionStr param provides the connection details to use the go SQL library.
 // It is empty in the following cases:
 //  - Driver is DynamoDB or a dump file mode.
 //  - This function is called as part of the legacy global CLI flag mode. (This string is constructed from env variables later on)
 // When using source-profile, the sqlConnectionStr is constructed from the input params.
 func SchemaConv(driver, sqlConnectionStr, targetDb string, ioHelper *IOStreams, schemaSampleSize int64) (*internal.Conv, error) {
+	conv := internal.MakeConv()
+	conv.TargetDb = targetDb
```

Comment:
Declaring conv here seems a little inconsistent. We should either: do it here and pass it through params in schemaFromDump codepath as well instead of declaring a new one again inside that. Or we should move this declaration inside schemaFromDatabase. Im leaning towards the former approach where we make similar changes for schemaFromDump.

Reply:
Suggest we move the creation of conv into schemaFromDatabase .

Reply:
Done